passes/techmap/muxcover.cc

```diff
@@ -330,13 +330,15 @@ struct MuxcoverWorker
 			ok = ok && follow_muxtree(S4, tree, bit, "BBS");
 
 			if (nodecode)
-				ok = ok && S1 == S2 && S2 == S3 && S3 == S4;
+				ok = ok && (S1 == S2 || S1 == State::Sx || S2 == State::Sx) &&
+					/*S2 == S3 && */
+					(S3 == S4 || S3 == State::Sx || S4 == State::Sx);
```

Comment:
That doesn't look right at all. The original code checks that S1, S2, S3, and S4 are all the same. The new code would be satisfied if only S1==S2 and S3==S4, but the link between S1 and S4 is broken. I'll prepare an alternative PR.

Reply:
My line of thinking was that we only check if S1==S2 and S3==S4 here, because we check T1==T2 later, and if T1 or T2 is don't care then there is no need for S2 == S3?